src/Api/SecretManagerFeatures/Models/Request/ProjectUpdateRequestModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.Entities;
+using Bit.Core.Utilities;
+
+namespace Bit.Api.SecretManagerFeatures.Models.Request
+{
+    public class ProjectUpdateRequestModel
+    {
+        [Required]
+        [EncryptedString]
+        public string Name { get; set; }
+
+        public Project ToProject(Guid id)
+        {
+            return new Project()
+            {
+                Id = id,
+                Name = this.Name,
+                DeletedDate = null,
```

Comment:
Do we need to set this here?

Reply:
We don't, good call. Thank you, I removed this line.